packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -376,7 +379,9 @@ export class PayPalClient {
     if (resultObj.ACK === 'Success' || resultObj.ACK === 'SuccessWithWarning') {
       return resultObj;
     } else {
-      throw new PayPalClientError(result.text, resultObj);
+      const err = new PayPalClientError(result.text, resultObj);
+      this.log.error('paypalClient.doRequest', err);
+      throw err;
```

Comment:
I think we can simplify this patch by reverting the `this.log` stuff? Else update the `doRequest` error tests to assert that it is now called with what arguments.